outputs.tf

```diff
@@ -1,71 +1,89 @@
 output "broker_id" {
-  value = "${join("", aws_mq_broker.default.*.id)}"
+  value       = "${join("", aws_mq_broker.default.*.id)}"
+  description = "The AmazonMQ broker ID"
```

Comment:
-The? For consistency

Reply:
If we removing `The` need to remove it also from `output "broker_arn"`